File: Curso_C#/Proyecto_Backend/MindFit_Intelligence_Backend/MindFit_Intelligence_Backend/Models/Equipamiento.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MindFit_Intelligence_Backend.Models
{
    public class Equipamiento
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int IdEquipamiento { get; set; }

        [Column(TypeName = "VARCHAR(50)")]
        public required string NombreEquipo { get; set; }

        [Column(TypeName = "DECIMAL(18,2)")]
        public decimal CostoAdquisicion { get; set; }

        [Column(TypeName = "DECIMAL(18,2)")]
        public decimal? PesoFijoKg { get; set; }
    }
}

File: Curso_CSharp/Otras Versiones/V9_Proyecto_MindFit_Intelligence_PreModuloIA/Proyecto_MindFit_Intelligence/Frontend/src/types/equipamiento.ts
export interface EquipamientoDto {
  idEquipamiento: number
  nombreEquipo: string
  costoAdquisicion: number
  pesoFijoKg: number | null
}

export interface EquipamientoInsertDto {
  nombreEquipo: string
  costoAdquisicion: number
  pesoFijoKg: number | null
}

export interface EquipamientoUpdateDto {
  nombreEquipo: string
  costoAdquisicion: number
  pesoFijoKg: number | null
}
